README.md

```diff
@@ -1,6 +1,6 @@
 Ptime — POSIX time for OCaml
 ============================
-%%VERSION%%
```

Comment:
I'm wondering, wouldn't [`dune subst`](https://dune.readthedocs.io/en/stable/usage.html#dune-subst) be able to handle these?

Reply:
Yes, but it's not run when used with `opam-monorepo`.

Reply:
... but it would when `dune-release` is executed to prepare the tarball that would be used by opam-monorepo. So yes that should be ok.

Reply:
If just vendoring via `(vendored_dirs)` it's probably not necessary to do the substitution of `%%VERSION%%` anyway.

Reply:
This change is here because I checked in the output of running `caml pkg/pkg.ml build --dev-pkg true`.